src/game/gameflow.c

```
@@ -219,6 +280,10 @@ static bool GameFlow_LoadLevelSequence(
     GAMEFLOW_SEQUENCE *seq = g_GameFlow.levels[level_num].sequence;
     int32_t i = 0;
     while (jseq_elem) {
+        const char *tmp_s;
+        GAMEFLOW_DISPLAY_PICTURE_DATA *data;
```

Let's keep this particular variable inside the `case` expression. The compiler will error out, but we can suppress the error by adding curly braces around the case like so:
```c
case foo: {
    int32_t local_var = 0;
    break;
}
```